doc/orders.md

```diff
@@ -0,0 +1,194 @@
+# Orders
+
+iOS examples are in Swift, and Android examples are in Kotlin.
+
+- [Fetch Claimed Orders](#fetch-claimed-orders)
+- [Fetch Unclaimed Orders](#fetch-unclaimed-orders)
+- [Observe Orders](#observe-orders)
+- [Claim Orders](#claim-orders)
+- [Update Orders](#update-orders)
+
+## <span id="fetch-claimed-orders">Fetch Claimed Orders</span>
+
+Fetch the latest claimed orders with the server. An order is claimed if it is associated with the current customer.
+
+##### iOS
+
+```swift
+FlyBuy.orders.fetch()
+```
+
+##### Android
+
+```kotlin
+fun fetchOrders() {
+    FlyBuy.orders.fetch { orders, sdkError ->
+        // Handle orders or deal with error
+    }
+}
```

Comment:
I don't think we need the outer function here, just the method call (to keep it consistent with swift code above)